extensions/aws/s3/MultipartUploadStateStorage.h

```diff
@@ -41,13 +41,13 @@ struct MultipartUploadState {
     : upload_id(upload_id),
       part_size(part_size),
       full_size(full_size),
-      upload_time(upload_time) {}
+      upload_time(upload_time.Millis()) {}
   std::string upload_id;
   size_t uploaded_parts{};
   uint64_t uploaded_size{};
   uint64_t part_size{};
   uint64_t full_size{};
-  Aws::Utils::DateTime upload_time;
+  int64_t upload_time;
```

Comment:
I think we should keep it in a type that specifies the format and meaning. Whether that's Aws DateTime, std::chrono::time_point or anything else doesn't really matter. Or if we stay on int, then please document what the integer represents. (Unix timestamp, but in milliseconds instead of seconds?)